plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -815,14 +815,17 @@ void QtOSGViewer::_CreateDockWidgets()
 
 void QtOSGViewer::_OnObjectTreeClick(QTreeWidgetItem* item,int num)
 {
-    RobotBasePtr robot;
     KinBodyPtr kinbody;
     KinBody::LinkPtr link;
 
     std::string mass;
 
-    //  Select robot in Viewers
-    _posgWidget->SelectItemFromName(item->text(0).toAscii().data());
+    //  Select kinbody in Viewers
+    {
+        QTreeWidgetItem* itemKinBody = item;
+        for (;!!itemKinBody->parent();) itemKinBody = itemKinBody->parent();
```

Comment:
although i appreciate squeezing everything in one line, can you convert this into:
```
while(!!itemKinBody->parent()) {
    itemKinBody = itemKinBody->parent();
}
```

thanks